kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -299,15 +312,25 @@ window.UserView = BaseView.extend({
                 this.totalPointViewXs = new TotalPointView({model: this.model, el: "#points-xs"});
             }
         } else {
+            var options = {};
+            var next = getParamValue("next");
+            var login = getParamValue("login");
+            if (login || this.login_start_open) {
+                options.start_open = true;
+
+                delete this.login_start_open;
```

Comment:
I... think this got left in from another approach I took to this. I will delete this flag.